src/main.ts

```diff
@@ -573,7 +585,7 @@ export default class WhatsApp {
 
   private saveKeys() {
     writeFile(
-      this.keysPath!,
+      this.parameters.keysPath!,
```

Comment:
```suggestion
      this.opts.keysPath!,
```
Update property again